pkg/cluster/provider.go

```diff
@@ -169,12 +169,9 @@ func ProviderWithPodman() ProviderOption {
 }
 
 // Create provisions and starts a kubernetes-in-docker cluster
-// TODO: move name to an option to override config
-func (p *Provider) Create(name string, options ...CreateOption) error {
+func (p *Provider) Create(options ...CreateOption) error {
```

Comment:
This breaks an API contract, and there are some folks that use `kind` as a library to create clusters, so I don't think this is a good idea. We need to keep backward compatibility so we don't break any consumers of this, so we either need to deprecate the current `Create` call, and introduce something new like `CreateWithOpts` or something along those lines. Or I suppose basically the same, but don't deprecate the `Create` call and just provide two different ways to do this.

Reply:
@stmcginnis you are right. I will fix it.

Reply:
Why are we breaking this?